hera_cal/vis_clean.py

```diff
@@ -692,14 +915,39 @@ def fourier_filter(self, filter_centers, filter_half_widths, mode,
                                 skipped[:, i] = True
                             elif dim == 1:
                                 skipped[i] = True
-
+            # A redundant system (in the airline sense of the word).
+            # just in case any artifacts make it through after our best flagging
```

Comment:
I don't see any reason why this should be in the Fourier filter method. For modularity, this should be its own method, not something inherent to this function.

I would advise to make a new module level function that operates directly on the data containers. a drive script can simply call this additional method if it likes, no reason to have the `model_rms_threshold` kwarg in fourier_filter

Reply:
I've broken this out into `vis_clean.flag_model_rms`